kotlinx-coroutines-core/common/src/Dispatchers.common.kt

```diff
@@ -78,4 +78,14 @@ public expect object Dispatchers {
      * the value of [CoroutineStart.UNDISPATCHED].
      */
     public val Unconfined: CoroutineDispatcher
+
+    /**
+     * Shuts down the dispatcher.
+     *
+     * Its implementation depends on the platform:
+     * - On Kotlin/JVM and Kotlin/JS takes no effect.
+     * - On Kotlin/Native requests termination of the worker created by the [Default] dispatcher.
+     */
+    @ExperimentalCoroutinesApi
+    public fun CoroutineDispatcher.shutdown()
```

Comment:
The signature seems to be not really usable because it has two receivers and also has effect _only_ on the default dispatcher in K/N which is a bit weird.

I'd suggest sticking with `Dispatchers.shutdownDefaultDispatcher` while we don't have a separate type for `Dispatchers.Default` and cannot do `Dispatchers.Default.shutdown()`.